src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -107,6 +111,131 @@ typealias Gold_Data_Ref_Spec = GoldInternal1_Spec
 typealias Gold_Alias = GoldInternal1
 typealias Gold_Alias_Spec = GoldInternal1_Spec
 
+class Gold_QCollection() : Entity {
+
+    override var internalId = ""
+
+    var name = ""
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+    var age = 0.0
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+    var lastCall = 0.0
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+
+    constructor(
+        name: String = "",
+        age: Double = 0.0,
+        lastCall: Double = 0.0
+    ) : this() {
+        this.name = name
+        this.age = age
+        this.lastCall = lastCall
+    }
+
+    fun copy(
+        name: String = this.name,
+        age: Double = this.age,
+        lastCall: Double = this.lastCall
+    ) = Gold_QCollection(
+        name = name,
+        age = age,
+        lastCall = lastCall
+    )
+
+    fun reset() {
+        name = ""
+        age = 0.0
+        lastCall = 0.0
+    }
+
+    override fun equals(other: Any?): Boolean {
+      if (this === other) {
+        return true
+      }
+
+      if (other is Gold_QCollection) {
+        if (internalId != "") {
+          return internalId == other.internalId
+        }
+        return toString() == other.toString()
+      }
+      return false;
```

Comment:
Indentation is only 2 here, where it should be 4

Reply:
I believe I've fixed the indentation by 2 that's in schema2kotlin (and updated the goldens)